app/views/trips/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Let's edit that trip!!!</h2>
+
+<%= form_for @trip do |f| %>
+  <%= f.label :rating %>
+  <%= f.number_field :rating, in: 1.0..5.0 %>
+
```

Comment:
I didn't know you can put a min and max on the number field - this is really cool!